src/features/offer/pages/Offer/Offer.tsx

```diff
@@ -71,15 +74,15 @@ export const Offer: FunctionComponent = () => {
     }
   })
 
+  const roundedPosition: Position = useMemo(() => {
+    return {
+      latitude: Number(userPosition?.latitude.toFixed(3)),
+      longitude: Number(userPosition?.longitude.toFixed(3)),
+    }
+  }, [userPosition?.latitude, userPosition?.longitude])
```

Comment:
[suggestion] : J'utiliserai une déstructuration vu que c'est répété deux fois mais c'est pas obligé

Reply:
Why not